clients/gtest/CMakeLists.txt

```diff
@@ -109,6 +109,16 @@ else( )
   target_link_libraries( rocblas-test PRIVATE ${HIPHCC_LOCATION} )
 endif( )
 
+set( ROCBLAS_TEST_DATA "${PROJECT_BINARY_DIR}/staging/rocblas_gtest.data")
```

Comment:
This generates the rocblas_gtest.data file from the gtest.yaml file. Once built, it doesn't need to be rebuilt unless gtest.yaml, one of its included yaml files, or the rocblas_gentest.py script is changed.